packages/react/src/component-tests/IcPopoverMenu/IcPopoverMenuData.tsx

```diff
@@ -183,7 +183,6 @@ export const PopoverMenuWithVariants = () => {
             label="Edit"
             description="This is used to edit the original."
           />
-          <IcMenuItem label="Show/Hide" variant="toggle" />
```

Comment:
What was the reason for removing this menu item?

Reply:
https://github.com/mi6/ic-ui-kit/issues/2722 - nested interactive elements are bad practice (and a failure in Accessibility Insights)

Reply:
I think i'm going to change this PR so that it doesn't remove the examples and leaves the work to that new ticket. As you've said on internal systems having role='menucheckboxitem' is allowed, we must just need to tweak our components